tests/integration/hive_metastore/test_grant_detail.py

```diff
@@ -25,7 +35,7 @@ def _deployed_schema(runtime_ctx) -> None:
 
 
 @retried(on=[NotFound, TimeoutError], timeout=dt.timedelta(minutes=3))
-def test_all_grant_types(runtime_ctx: MockRuntimeContext, _deployed_schema: None):
+def test_all_grant_types(runtime_ctx: MockRuntimeContext, sql_backend_tacl: SqlBackend, _deployed_schema: None):
```

Comment:
I am mimicking the assessment workflow here:
1. `TablesCrawler` and `UdfsCrawler` run against the "regular" backend
2. `GrantsCrawler` run against the "tacl" backend